lib/restforce/middleware/caching.rb

```diff
@@ -18,7 +18,7 @@ def cache_key(env)
     end
 
     def use_cache?
-      !@options.key?(:use_cache) || @options[:use_cache]
+      @options.fetch(:use_cache, true)
```

Comment:
Good move here - if I'd written this I'd have use `#fetch` but didn't fix these when I did my refactor.